hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapFetchIndexOperation.java

```diff
@@ -352,9 +351,14 @@ protected void writeInternal(ObjectDataOutput out) throws IOException {
         out.writeInt(sizeLimit);
     }
 
+    @Override
+    public int getFactoryId() {
+        return MapDataSerializerHook.F_ID;
+    }
```

Comment:
Minor: This is the same method definition as in the base MapOperation and can be removed